src/WebGL.ts

```diff
@@ -984,34 +987,40 @@ export class WebGLRenderingContext {
 	}
 
 	@inline getPreserveDrawingBuffer(): GLboolean {
-		getPreserveDrawingBuffer(this.gl_id);
+		return getPreserveDrawingBuffer(this.gl_id);
 	}
 
 	@inline setPreserveDrawingBuffer(value: GLboolean): void {
 		setPreserveDrawingBuffer(this.gl_id, value);
 	}
 
 	@inline getSize(): GLint {
+    // @ts-ignore
 		return getSize(this.gl_id);
 	}
 
 	@inline getType(): GLenum {
+    // @ts-ignore
```

Comment:
This sliences TS errors like `Argument of type 'number' is not assignable to parameter of type 'externref'. ts(2345)`.

Is the return correct? Or should it be `externref`?